src/sonic-yang-models/doc/Configuration.md

```diff
@@ -1487,6 +1488,24 @@ name as object key and member list as attribute.
 }
 ```
 
+### Tunnel
+
+This table configures the MUX tunnel for Dual-ToR setup
+```
+{
+    "TUNNEL": {
+        "MuxTunnel0": {
+            "dscp_mode": "uniform",
+            "dst_ip": "10.1.0.32",
+            "ecn_mode": "copy_from_outer",
+            "encap_ecn_mode": "standard",

```

Comment:
please also include the encap and decap qos maps related attributes. I see that they have been added to the actual yang definition